### ISSUE TITLE
Change logic that determines object status

### ISSUE THREAD

**Issue by blalbrit** [sul-dlss/dor-services#221]
the current logic is that the latest lifecycle wins … but maybe it should be earliest instance of "registered” then the latest instance of of submitted/accessioned/opened

per @LynnMcRae

**Comment by LynnMcRae** [comment]
This would correct an instance where something like dpgImageWF, if added later with a new lifecycle=registered, would not pre-empt the original lifecycle=registrered.

**Comment by atz** [comment]
@LynnMcRae You are not intending to differentiate the typo'd version of `resistrered`, right?

**Comment by LynnMcRae** [comment]
nope, "registered" by any other name should not exist anywhere but in my hasty typing sans spell-check.

**Comment by ndushay** [comment]
@peetucket did PR #644   or @jmartin-sul  did your PR  address this ancient ticket?

**Comment by jmartin-sul** [comment]
@ndushay, i don't recall putting up any PRs that addressed this issue.

**Comment by ndushay** [comment]
sorry @jmartin-sul i probably garbled something or other in my mind.  I've been looking at a lot of repos and issues and PRs and it all runs together sometimes.

**Comment by jmartin-sul** [comment]
totally understandable, @ndushay!  i had to go look at my closed dor-services PRs to make sure i hadn't forgotten something myself 🙂
